frontend/src/styles.css

```diff
@@ -1,3 +1,4 @@
 /* You can add global styles to this file, and also import other style files */
 @import "primeng/resources/themes/aura-dark-blue/theme.css";
-@import "primeng/resources/primeng.css";
\ No newline at end of file
+@import "primeng/resources/primeng.css";
+@import "primeflex/primeflex.css";
```

Comment:
Should this be `@import "primeflex/primeflex.scss";` instead of `@import "primeflex/primeflex.css";`?

I'm getting an error with the current one and only works after I change to `.scss`

`.scss` seems to match with their documentation too

![image](https://github.com/user-attachments/assets/6dafbc0a-d2ed-4aac-a05f-3498086d312b)

Reply:
Ok, I've changed it to .scss and it works on my side as well.